packages/console/src/browser/console-widget.ts

```diff
@@ -26,6 +26,7 @@ import { ConsoleHistory } from './console-history';
 import { ConsoleContentWidget } from './console-content-widget';
 import { ConsoleSession } from './console-session';
 import { ConsoleSessionManager } from './console-session-manager';
+import * as Monaco from '@theia/monaco-editor-core';
```

Comment:
@tsmaeder, I've pushed a change so that all imports of `@theia/monaco-editor-core` are `import * as monaco from '@theia/monaco-editor-core';`. It does shrink the diff considerably.